docker-compose-installed.yaml

```diff
@@ -69,3 +77,14 @@ services:
     depends_on:
       - validator
     entrypoint: sawtooth-sabre -vv --connect tcp://validator:9704
+
+  sabre-cli:
```

Comment:
This compose file already has a section for building a CLI container: https://github.com/hyperledger/sawtooth-sabre/blob/82d68a7f16bdf408435baa11ed088919f0cfb507/docker-compose-installed.yaml#L54. Perhaps you could add the volumes up there?

Reply:
Thanks for looking into the PR @rbuysse . I'll push another commit that changes the image for shell instead of moving the volume, and we can see how it looks.

To motivate it: one goal was to realign the two docker-compose files (default and installed) and be able to run the tutorial by swapping the docker-compose files when starting the network, one benefit being to shortcircuit the various cargo builds at startup https://sawtooth.hyperledger.org/docs/sabre/releases/latest/application_developer_guide.html#starting-the-network
then all the sections of the tutorial to do cr, upload, ns, and perm, that are preconditioned with  "Enter the sabre-cli docker container" would still work as is, that's why I put the volumes in the sabre-cli, and not shell.
Looking at the PR again, to follow through the original motivation, I would base the shell container not on `cli/Dockerfile-installed` but on `hyperledger/sawtooth-all:1.0` like in the docker-compose.yml. That would be one more realignment. In the tutorial, the shell container is only used for intkey related commands, so sawtooth-all is sufficient for that.
The added volumes to the `validator` and `sabre-cli` are for sharing the keys between containers. I noticed the docker-compose-installed.yaml was free of any volumes and this PR breaks this property, and could not tell if there was a best way to break it or not.